internal/state/config_process.go

```
@@ -57,7 +62,48 @@ func NewGroundControlConfigEvent(states []string) scheduler.Event {
+		return fmt.Errorf("failed to decode response: %v", err)
+	}
+
+	log.Info().Msgf("Received %d states from Ground Control", len(payload.States))
+
+	if err := config.UpdateStates(payload.States); err != nil {
+		return fmt.Errorf("failed to update states in config: %v", err)
+	}
+
+	event := NewGroundControlConfigEvent(payload.States)
+	if f.eventBroker != nil {
+		f.eventBroker.Publish(event, ctx)
```

golangci-lint (1.64.8)

Error return value of `f.eventBroker.Publish` is not checked

(errcheck)